lib/commands/new/loaders/require.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const ProjectItem = require('../../../project-item').ProjectItem;
+
+module.exports = function(project) {
+  project.loader = project.model.loader.id;
+  project.model.loader = undefined; // remove loader from model as it is actually a property of model.build 
```

Comment:
The model is setup based on the result from WorkflowEngine [here](https://github.com/aurelia/cli/pull/631/files#diff-90607133b006fdb0120c6a67bb7ae68aR26). I remove that property so there is no loader node at the top level of `aurelia.json`. Make sense?